docker-compose.yml

```diff
@@ -25,8 +25,6 @@ services:
       POSTGRES_PASSWORD: $HACKTOBERFEST_DATABASE_PASSWORD
     volumes:
       - db_data:/var/lib/postgresql/data
-    ports:
-      - "5432:5432"
```

Comment:
Could we leave this in? It allows me to have RubyMine access the DB directly w/ a nice UI if I need to debug things :P